README.local-opsworks.md

```diff
@@ -32,52 +32,68 @@ We do not currently support all-in-one nodes, but that should be coming soon.
 * Start your environment! This takes about 20 to 25 minutes on my beefy machine
   with SSD.
 
+        # Start an all-in-one node on http://10.10.10.50/
+        ./bin/all_in_one up
+        # Start a multi-node cluster on http://10.10.10.10/
```

Comment:
Yes it does!  

FWIW, that'd be possible without all the tooling I wrote in the `local_cluster` and `all_in_one` scripts, but I think they should make things easier over all.